test/runtime/Policies.go

```diff
@@ -669,6 +671,239 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
+			[{
+				   "endpointSelector": {"matchLabels":{"%s":""}},
+				   "ingress": [{
+					   "fromEndpoints": [
+					   {"matchLabels":{"%s":""}}
+				   ]
+				   }, {
+				   "fromCIDR": [
+					   "%s",
+					   "%s"
+				   ]
+				   }]
+			   }]`, httpd1Label, httpd2Label, ipv4Prefix, ipv6Prefix)
+		renderAndImportPolicy(script)
+
+		By(fmt.Sprintf("Pinging httpd1 IPv4 from app3 (should work because we allow ingress from CIDR %s which app3 is included)", ipv4Prefix))
+		res = vm.ContainerExec(helpers.App3, helpers.Ping(helpers.Httpd1))
+		res.ExpectSuccess("Unexpected failure pinging %s IPv4 from %s: %s", helpers.Httpd1, helpers.App3, res.CombineOutput().String())
+
+		By(fmt.Sprintf("Pinging httpd1 IPv6 from app3 (should work because we allow ingress from CIDR %s which app3 is included)", ipv6Prefix))
+		res = vm.ContainerExec(helpers.App3, helpers.Ping6(helpers.Httpd1))
+		res.ExpectSuccess("Unexpected failure pinging %s IPv6 from %s: %s", helpers.Httpd1, helpers.App3, res.CombineOutput().String())
+
+		vm.PolicyDelAll().ExpectSuccess("Unable to delete all policies")
+
+		// Make sure that combined label-based and CIDR-based policy works.
+		By(fmt.Sprintf("Importing Policy Allowing Ingress From %s --> %s And From CIDRs %s", helpers.Httpd2, helpers.Httpd1, ipv4OtherNet))
+		script = fmt.Sprintf(`
+			[{
+				   "endpointSelector": {"matchLabels":{"%s":""}},
+				   "ingress": [{
+					   "fromEndpoints": [
+					   {"matchLabels":{"%s":""}}
+				   ]
+				   }, {
+				   "fromCIDR": [
+					   "%s"
+				   ]
+				   }]
+			   }]`, httpd1Label, httpd2Label, ipv4OtherNet)
+		renderAndImportPolicy(script)
+
+		By(fmt.Sprintf("Pinging httpd1 IPv4 from app3 (should NOT work beacuse we only allow traffic from %s to %s)", httpd2Label, httpd1Label))
+		res = vm.ContainerExec(helpers.App3, helpers.Ping(helpers.Httpd1))
+		res.ExpectFail("Unexpected success pinging %s IPv4 from %s: %s", helpers.Httpd1, helpers.App3, res.CombineOutput().String())
+
+		By(fmt.Sprintf("Pinging httpd1 IPv6 from app3 (should NOT work because we only allow traffic from %s to %s)", httpd2Label, httpd1Label))
+		res = vm.ContainerExec(helpers.App3, helpers.Ping6(helpers.Httpd1))
+		res.ExpectFail("Unexpected success pinging %s IPv6 from %s: %s", helpers.Httpd1, helpers.App3, res.CombineOutput().String())
+
+		vm.PolicyDelAll().ExpectSuccess("Unable to delete all policies")
+
+		By("Testing CIDR Exceptions in Cilium Policy")
+		By(fmt.Sprintf("Importing Policy Allowing Ingress From %s --> %s And From CIDRs %s Except %s", helpers.Httpd2, helpers.Httpd1, ipv4Prefix, ipv4PrefixExcept))
+		script = fmt.Sprintf(`
+			[{
+				"endpointSelector": {"matchLabels":{"%s":""}},
+				"ingress": [{
+					"fromEndpoints": [
+						{"matchLabels":{"%s":""}}
+					],
+					"fromCIDRSet": [ {
+						"cidr": "%s",
+						"except": [
+							"%s"
+						]
+					}
+					]
+				}]
+			}]`, httpd1Label, httpd2Label, ipv4Prefix, ipv4PrefixExcept)
+		renderAndImportPolicy(script)
+
```

Comment:
Something is missed here no?

Reply:
I don't understand what you mean here. What is "something"?